pkg/reconciler/sequence/controller.go

```diff
@@ -42,14 +46,25 @@ func NewController(
 
 	sequenceInformer := sequence.Get(ctx)
 	subscriptionInformer := subscription.Get(ctx)
+	serviceaccountInformer := serviceaccountinformer.Get(ctx)
+
+	featureStore := feature.NewStore(logging.FromContext(ctx).Named("feature-config-store"))
+	featureStore.WatchConfigs(cmw)
```

Comment:
Actually we need to reconcile the Sequence again, if the configmap changes. We have done this in other resources with the `globalResync` func. E.g.:

https://github.com/knative/eventing/blob/a9320dc1358264317bdf540ad2765e42623d0311/pkg/reconciler/subscription/controller.go#L53-L60

and

https://github.com/knative/eventing/blob/a9320dc1358264317bdf540ad2765e42623d0311/pkg/reconciler/subscription/controller.go#L76-L78

Can you include this here too?

Reply:
done, ptal!